x-pack/legacy/plugins/alerting/README.md

```diff
@@ -59,7 +59,7 @@ This is the primary function for an alert type. Whenever the alert needs to exec
 
 ### Example
 
-This example receives server and threshold as parameters. It will read the CPU usage of the server and fire actions if the reading is greater than the threshold.
+This example receives server and threshold as parameters. It will read the CPU usage of the server and schedule actions to be fired (by enqueueing them with the task manager) if the reading is greater than the threshold.
```

Comment:
I'm thinking we could remove the fire and enqueue terminology here and use something like: `and schedule actions to be executed`.

Reply:
yup - we changed the action verb to `execute`

Reply:
coolio

Reply:
How does `This example receives server and threshold as parameters. It will read the CPU usage of the server and schedule actions to be executed (asynchronously by the task manager) if the reading is greater than the threshold.` sound?

Reply:
Love it!